portality/view/doaj.py

```diff
@@ -163,7 +163,7 @@ def autocomplete(doc_type, field_name):
 
 @blueprint.route("/toc")
 def list_journals():
-    js = models.Journal.all_in_doaj(page_size=1000, minified=True)
```

Comment:
Two things for me:

1. it was not being used anywhere else
2. it did not return model objects - just JSON ones with a few values (the point of minified). We actually need to use a method in the template serving this endpoint, `.toc_id()`, so in fact we do end up needing the whole object. I just didn't see any point in it after that - but I may have missed something!